tracer/src/Datadog.Trace/DuckTyping/DuckType.Utilities.cs

```diff
@@ -39,7 +39,7 @@ private static void EnsureArguments(Type proxyType, object instance)
         /// </summary>
         /// <param name="builder">Module builder</param>
         /// <param name="type">Type to gain internals visibility</param>
-        private static void EnsureTypeVisibility(ModuleBuilder builder, Type type)
+        internal static void EnsureTypeVisibility(ModuleBuilder builder, Type type)
```

Comment:
Internal because previously I was using this only for ducktyping but now is being used by the Activities namespace. We can leave like this, or the "right way of doing it" is refactoring this to a helper class in Utils namespace or something, although Ducktype will not be isolated anymore :/ . I kind of liked having Ducktyping isolated in that namespace, but in the other hand all the assembly is depending on Ducktyping so... 🤷🏻 . Not strong opinion, If you have one I can do the refactor.

Reply:
No, sounds fine to me 👍